content/html/concepts/semantic-html/terms/summary/summary.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '<summary>'
+Description: 'Used as a label for the <details> element and when clicked reveals additional information.'
```

Comment:
Could be we revise the latter part of the description to say "... and reveals additional information when clicked."?